langserver/workspace_refs.go

```diff
@@ -67,6 +67,7 @@ func (h *LangHandler) handleWorkspaceReferences(ctx context.Context, conn JSONRP
 			}
 		}
 		unvendoredPackages[bpkg.ImportPath] = struct{}{}
+		unvendoredPackages[bpkg.ImportPath+"_test"] = struct{}{}
```

Comment:
Right, so if you find the actual usage of `unvendoredPackages` below 86-90 you'll see that this is a map of packages that we're interested in scanning for external references (b/c `AfterTypeCheck` is invoked for transitive dependencies too). A better name _might_ be "interestingUnvendoredPackages" but that's a bit lengthy already, hence leaving it as-is.